src/test/regress/expected/columnar_memory.out

```diff
@@ -18,10 +18,13 @@ INSERT INTO t
         -- sample memusage instead of recording everyr row for speed
         CASE WHEN i % 100 = 0 THEN columnar_test_helpers.top_memory_context_usage() ELSE 0 END
  FROM generate_series(1, 49999) i;
+DEBUG:  not pushing down function calls in a multi-statement transaction
```

Comment:
can we remove this debug message? it seems a bit out of wack

Reply:
or should we make it DEBUG4 or such?